examples/gallery/lines/great_circles.py

```diff
@@ -0,0 +1,43 @@
+"""
+Generate points along great circles
+-----------------------------------
+
+The :meth:`pygmt.project` method can generate points along a great circle
+whose center and end points can be defined via the ``center`` and ``endpoint``
+parameters, respectively. Using the ``generate`` parameter allows to generate
+(*r*, *s*, *p*) points every *dist* units of *p* along a profile as
+output. By default all units (*r*, *s* and *p*) are set to degrees while
+``unit=True`` allows to set the unit for *p* to km.
```

Comment:
Using `unit=True` seems very imprecise. Maybe we should use `unit="k"`, and point out the other units available at https://docs.generic-mapping-tools.org/6.3/project#units?

Reply:
> Maybe we should use unit="k", and point out the other units available at docs.generic-mapping-tools.org/6.3/project#units?

But `unit` is an alias of `-Q` option, which can't accept unit strings.

Reply:
Ah ok, I misread the `-Q` option. Please ignore this then.